### PR TITLE
Update core repo to resolve absolute paths for dependencies, update c…

### PR DESCRIPTION
**Pull request by Davoy**
These changes removes absolute paths for dependencies in order to have a more dynamic installation process. Some configuration files have been added with trial API keys for testing and additional required data.

Also resolved legacy code in wttr.py when calling the wego terminal command, it was being supplied invalid/deprecated flags. Flags have been updated or removed.

Lastly, geo-proxy.py cache had no check if the cache directory exists before writing, so I added validation on the directory to be created if not already present.

### CODE REVIEW

File: lib/airports.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import csv
+import os
 
-AIRPORTS_DAT_FILE = '/home/igor/wttrin-geo/share/airports.dat'
+AIRPORTS_DAT_FILE = '{}/airports.dat'.format(os.getenv('WTTR_MYDIR'))
```
Comment (braunshedd) [inline lib/airports.py]: use inline formatting for python strings, i.e.
`f"{os.getenv('WTTR_MYDIR')}/airports.dat"`

Comment (braunshedd) [reply lib/airports.py]: I believe they're called "f-strings"

Comment (Davoy) [reply lib/airports.py]: Awesome! This has been updated

**Review by kmuga001 — APPROVED** [review]
This looks good to me! And thank you for making sure that the wego and pyphoon do not come from igor's home directory in the globals.py file.

**Pull request merged**